kong/clustering/wrpc_data_plane.lua

```diff
@@ -60,15 +63,11 @@ local function get_config_service()
     wrpc_config_service:import("kong.services.config.v1.config")
     wrpc_config_service:set_handler("ConfigService.SyncConfig", function(peer, data)
       if peer.config_semaphore then
-        if data.config.plugins then
-          for _, plugin in ipairs(data.config.plugins) do
-            plugin.config = protobuf.pbunwrap_struct(plugin.config)
-          end
-        end
-        data.config._format_version = data.config.format_version
-        data.config.format_version = nil
+        local json_config = assert(inflate_gzip(data.config))
+        peer.config_obj.next_config = assert(cjson_decode(json_config))
```

Comment:
Similar here comment, see (not sure if there should be `yielding` in somewhere else with `wrpc`):
https://github.com/Kong/kong/blob/master/kong/clustering/data_plane.lua#L252-L256